packages/core/src/Provider/StakePoolProvider/types/StakePoolProvider.ts

```diff
@@ -24,13 +24,23 @@ export interface MultipleChoiceSearchFilter<T> {
 
 /** Options for the fuzzy search on stake pool metadata */
 export interface FuzzyOptions {
+  /** Determines how close the match must be to the location. */
+  distance: number;
+
+  /** Determines approximately where in the text is the pattern expected to be found. */
+  location: number;
+
   /** Maximum threshold. `0`: exact match; `1`: match everything. */
   threshold: number;
 
   /** Weights of metadata fields. Free positive values: they will be normalized internally. */
   weights: { description: number; homepage: number; name: number; poolId: number; ticker: number };
 }
 
+export type PartialFuzzyOptions = Partial<Omit<FuzzyOptions, 'weights'>> & {
```

Comment:
If we follow the suggestion further down we can remove this type entirely.

Reply:
f51d482c9c should solve this